app/services/saml_request_validator.rb

```diff
@@ -48,10 +47,10 @@ def authorized_email_nameid_format
   end
 
   def email_nameid_format?
-    nameid_format == 'urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress'
+    nameid_format == 'urn:oasis:names:tc:SAML:2.0:nameid-format:emailAddress'
```

Comment:
Wdyt of changing this one up to look like this? It gives us backward compatibility.

```ruby
[
  'urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress',
  'urn:oasis:names:tc:SAML:2.0:nameid-format:emailAddress',
].include?(nameid_format)
```

Reply:
Yes, done.